### DISCUSSION TITLE
Olası depreme hazırlık - Bina durum sorgulama ve ihbar projesi

### DISCUSSION THREAD

**Discussion by alisaitteke** [acikkaynak/deprem-yardim-projesi#87] [Ideas]
Merhabalar.

Olası depremlere hazırlığın önemini anlatmaya malesef luzüm kalmadı. İlgili kurumlar yavaş çalışıyor ve bireysel başvurular önemsenmiyor. Ve malesef kendi evinin kolonunu kestiğini bile unutan insanlarla aynı binalarda yaşıyoruz.

Bu sebeple şeffaf, bağımsız ve devlet kurumları ile koordineli bir önlem alınması gerekiyor. Kısacası bir harita üzerinde bütün bir şehrin durumunu gösterebilirsek ilgili belediyelerin ve muhtarlıkların daha çok dikkatini çekeceğini düşünüyorum. Zira bina durumu iyi olan belediyelerin bölgeleri daha güvenli görüneceği için diğer belediyeleri / muhtarları pozitif olarak etkileyeceğini düşünüyorum. Belirli aralıklarla oluşturulacak basın kiti ile gerekli medya desteğini alabileceğimizden de eminim.

Yani;
Her binanın/sokağın/mahallenin ve şehrin tek tek kontrol edilmesi, edilmeyenlerin görülebilmesi, edilmediği zaman kamuoyunun oluşturulması ve her ay infografik yayınlar yapılabilmek için bir mobil ve web tabanlı uygulama geliştirmek istiyorum. 

Ben ESRI, QGis, ARCGis ve data anlamlandırma teknolojileri üzerinde uzmanım. Bu projeyi ortak akılla ve en önemlisi bağımsız ve şeffaf yürütmek için burada tartışılmasını istedim. Kimse katılmazsa aşağıda ki teknoloji stack ine uzun yıllardır hakim olduğum için tek başıma girişmeyi düşünüyorum :)

Aklimda ki stack basitce:
- Firebase
- Flutter
- NuxtJS (Web UI) - SEO onemli oldugu icin serverside olması, Vue gibi basit bir dil kullanması, Cloudflare pages ta ucretsiz ve performansli calistigi icin tercih ediyorum. 

İhtiyaç duyulan tahmini iş gücü:

- Flutter (IOS, Android, belki mobile web) - 1 Kişi / X Gün
- Backend (Firebase veya AWS Lambda) - 1 Kişi / X Gün
- WEB UI - JS - 1 Kişi / X Gün
- WEB UI - CSS, HTML - 1 Kişi / X Gün
- Mernis entegrasyonu - 1 kişi / X Gün
- Belediyelerin entegrasyonu - 2 kişi + 1 PM / X Gün

**Comment by zgnruya** [comment]
Merhaba, 2020 yılında yaklaşık 1,5 sene Güven İstanbul adında mobil/web tabanlı bir deprem güvenliği platformu üzerine çalıştık. Bir sosyal girişim olarak kuluçka sürecinden geçtik vs.Ancak kurumların tavrı, veriye ulaşmada zorluk ve sürdürülebilir emek/ iş gücü eksikliği gibi sebeplerle nihayetinde dağıldık. Bugün ihtiyaç duyulan pek çok şeyin o gün platform içinde yer bulması planlanmıştı ne yazık ki... Bu platformu tekrar hayata geçirmeye çalışıyorum. Ancak yukarıda yazdığını mernis entegrasyonu ve belediyelerin entegrasyonu kısmında muhtemel problemler olacaktır ya da gerçekleşmesi zor görünüyor. Yine de sistem kurulur ve kullanım söz konusu olabilirse o yönde gelişebilir.. Neden olmasın? Daha detaylı görüşmek isterseniz ozgunruya@gmail.com üzerinden ulaşabilirsiniz bana, burayı sık kullanmıyorum.

**Reply by sercangoger** [reply]
eğer buradaki ekipler tarafından proje onaylanır ise mobil tarafa entegre yapabiliriz. şuan ekip kurulmakta ve superApp geliştirmesi için planlama yapılacaktır.

**Reply by zgnruya** [reply]
bununla ilgili süreci nereden takip edebilirim? @sercangoger

**Reply by sercangoger** [reply]
discord 'a gidebilirsiniz.orada bütün ekipleri görebileceksinizdir.
